src/wallet/gtest/test_orchard_wallet.cpp

```diff
@@ -70,6 +70,8 @@ TEST(OrchardWalletTests, TxContainsMyNotes) {
     // the key to the wallet; it should not be detected as ours.
     auto skNotOurs = RandomOrchardSpendingKey();
     auto tx2 = FakeOrchardTx(skNotOurs, libzcash::diversifier_index_t(0));
-    wallet.AddNotesIfInvolvingMe(tx2);
+    wallet.AddNotesIfInvolvingMe(0, tx2);
     EXPECT_FALSE(wallet.TxContainsMyNotes(tx2.GetHash()));
+
+    RegtestDeactivateNU5();
```

Comment:
Opened #5613 because I'm sure there's a better way to handle this in modern `googletest` than the hack we came up with in 2018.